packages/core-react/src/components/Select/index.js

```diff
@@ -2,6 +2,13 @@ import React from 'react';
 import clsx from 'clsx';
 import PropTypes from 'prop-types';
 
+export const InputIcon = ({ prepend, icon }) => {
```

Comment:
Nit: We can probably change the name of this component because it's not just for `Inputs` anymore. Also, feel free to move it out as a nested component. Otherwise, since it's a small thing here, I don't mind this being inside here too.